iocore/eventsystem/unit-tests/test_MIOBufferWriter.cc

```diff
@@ -213,5 +223,9 @@ TEST_CASE("MIOBufferWriter", "[MIOBW]")
 void
 _ink_assert(const char *a, const char *f, int l)
 {
+#if USE_EXCEPTION
   throw InkAssertExcept();
+#else
+  std::exit(1);
```

Comment:
Why not do an exit all the time?

Reply:
I can only do that if I remove the tests that purposely cause an ink_assert().  Is that what you want?

Reply:
If it's just for testing, could you tweak `_ink_assert` to set a global flag and check that after?

Reply:
@SolidWallOfCode that would not be valid because my code, as is to be expected, treats ink_assert() as [[noreturn]] (never returning).

My best guess is that the problem is somehow caused by the complex use of exceptions by catch.hpp.  I proposed trying set/longjmp() as a solution in another pull request, but Bryan rejected that.

Reply:
I mentioned to set a global before.  He would have to return from functions when the assert condition happens too.  How many tests test for asserts?

Reply:
There are two tests that cause ink_assert().

Reply:
I would rather not have throws or setjmp in the code.  If you can set a global and return that would be preferable.

Reply:
As I said, I don't see how I can.